bmwqemu.pm

```diff
@@ -155,7 +155,7 @@ sub _check_publish_vars {
         for my $type (qw(STORE PUBLISH FORCE_PUBLISH)) {
             my $name = $type . "_HDD_$i";
             next unless my $out = $vars{$name};
-            die "HDD_$i also specified in $name. This is not supported" if grep { $_ eq $out } @hdds;
+            die "HDD_$i also specified in $name. This is not supported" if grep { $_ && $_ eq $out } @hdds;
```

Comment:
Is this for the null backend?

Reply:
That's independant of the backend, part of commit https://github.com/os-autoinst/os-autoinst/pull/1360/commits/6b6182d6a807c01c2d273e735b3e6eb127c1bc02 which can happen whenever this method is called without initializing the full stack in before but not only in unit tests but also when calling `isotovideo` locally under certain circumstances.

Reply:
Aha. Thanks for clarifying!